### ISSUE TITLE
Kannaḍa text not rendering in thumbnails

### ISSUE THREAD

**Issue by cadecairos** [mozilla/api.webmaker.org#174]
Yet another language Blitline cannot render:

![](https://dgu16gle6my6q.cloudfront.net/mobile-center-cropped/small/webmaker-desktop/aHR0cHM6Ly9iZXRhLndlYm1ha2VyLm9yZy8jL3RodW1ibmFpbD91c2VyPTQ2NDI3NyZwcm9qZWN0PTEwNzg1JnBhZ2U9MjkyNjUmdD0xNDQwMDg1ODA4MjI5)

https://beta.webmaker.org/#/project?user=464277&project=10785

https://en.wikipedia.org/wiki/Kannada

cc/ @secretrobotron @jbuck

**Comment by cadecairos** [comment]
Thai is also not rendering

![](https://dgu16gle6my6q.cloudfront.net/mobile-center-cropped/small/webmaker-desktop/aHR0cHM6Ly9iZXRhLndlYm1ha2VyLm9yZy8jL3RodW1ibmFpbD91c2VyPTQ2MzExNiZwcm9qZWN0PTEwMDQ4JnBhZ2U9Mjc2NDMmdD0xNDQwMDg1ODA4MjI5)

https://beta.webmaker.org/#/project?user=463116&project=10048

**Comment by secretrobotron** [comment]
Yeah, I've seen a few of these. Pretty sure Thai doesn't render either.

Also saw a lot of improperly overlapping images while going through projects yesterday.
